.github/workflows/cuda/cu113-Linux-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+CUDA_HOME=/usr/local/cuda-11.3
+LD_LIBRARY_PATH=${CUDA_HOME}/lib64:${LD_LIBRARY_PATH}
+PATH=${CUDA_HOME}/bin:${PATH}
+
+export FORCE_CUDA=1
+export TORCH_CUDA_ARCH_LIST="3.5;5.0+PTX;6.0;7.0;7.5;8.0;8.6"
+export CUDA_HOME=/usr/local/cuda-11.3
```

Comment:
Reject this change, this is overkill. Just add validation at the beginning of the `model.py`.

Reply:
It's just a test. see the commit `fix: passby flash attention`. And I think no need to check this. Once import succeeds, they should match.

Reply:
https://github.com/jina-ai/clip-as-service/actions/runs/3459017298/jobs/5774137064